GlossaryOfTerms.ttl

```diff
@@ -3864,11 +3879,10 @@ owl:IrreflexiveProperty
     wdrs:describedby  source: ;
     schema:name "SPARQL" ;
     skos:altLabel "SPARQL" ;
-    schema:description """Recursive acronym of "SPARQL Protocol and RDF Query Language", SPARQL comprises
-                          a Declarative Structured Query language for RDF-model-based structured data, Query
-                          Results Serialization Formats, HTTP-based Query Service Protocol, and an HTTP-based
-                          Database- (or Store-) oriented protocol for Create, Read, Update, and Delete (CRUD)
-                          operations."""@en ;
+    schema:description """Recursive acronym that covers: a Declarative Structured Query language for
+                      RDF-model-based structured data, Query Results Serialization Formats,
+                      HTTP-based Query Service Protocol, and an HTTP-based Database- (or Store-)
+                      oriented protocol for Create, Update, and Delete (CRUD) operations."""@en ;
```

Comment:
This undoes a [previous commit from me](https://github.com/OpenLinkSoftware/glossaries/commit/cac9b5a2767189b21c2e144ef01b492d796ad95b) (applied by @openlink), which added the explicit expansion of the SPARQL acronym (which is not revealed by the current [old] text).